ui/static/ui/js/xml_panel.jsx

```diff
@@ -0,0 +1,70 @@
+define('xml_panel',
+  ['react', 'jquery'],
+  function (React, $) {
+  'use strict';
+
+  var XmlPanel = React.createClass({
+    render: function () {
+      return <div>
+          <textarea className="form-control textarea-xml"
+                    readOnly="true"
+                    value={this.state.contentXml}
+          />
+          <p className="text-right">
+            <a id="copy-textarea-xml" href="#"
+               className="btn btn-white"
+               onClick={this.selectXml}>Select XML</a>
+          </p>
+      </div>;
+    },
+
+    selectXml: function (event) {
+      event.preventDefault();
+      $('.textarea-xml').select();
```

Comment:
I know you just copied this from what it was before but can you do:

    $(React.findDOMNode(this)).find(".textarea-xml").select();

instead? It would limit the scope to items within the React component.